add initial version of test-protocol template (#303)
kyma-project/application-connector-manager: .github/ISSUE_TEMPLATE/test-protocol.md

---
name: Test protocol
about: Document a manual testing cycle
---

# Test details

* Tested ACM Module Version: {e.g. 1.1.5}

> * Github release: {Link to GH release}

Reviewer: ```suggestion
* GitHub release: {Link to GH release}
```

* Test execution date: {2025-07-29}

> * Tester: {@your-Github-user}

Reviewer: ```suggestion
* Tester: {@your-GitHub-user}
```

# Test protocol

The detailed test flow is explained in our internal documentation.

## Pre-requisites

- [ ] Configure System Landscape and Formations in BTP Cockpit

> - [ ] Access to Back Office and Storefront possible

Reviewer: ```suggestion
- [ ] Access to back-office and storefront possible
```
based on [this doc](https://doc.oroinc.com/user/concept-guides/administration/menus/) this is how it should be spelled

## Test flow:

> ### 1. Register events in Back office

Reviewer: ```suggestion
### 1. Register events in back-office
```

 - [ ] Passed
 - [ ] Failed
 

>      Link to Github issue: {GH link}

Reviewer: ```suggestion
     Link to GitHub issue: {GH link}
```

> ### 2. Prepare serverless function in Kyma

Reviewer: ```suggestion
### 2. Prepare Serverless Function in Kyma
```

 - [ ] Passed
 - [ ] Failed

>      Link to Github issue: {GH link}

Reviewer: ```suggestion
     Link to GitHub issue: {GH link}
```

### 3. Trigger an event in EC

 - [ ] Passed
 - [ ] Failed

>      Link to Github issue: {GH link}

Reviewer: ```suggestion
     Link to GitHub issue: {GH link}
```

### 3. Verify event was delivered and API called

 - [ ] Passed
 - [ ] Failed

>      Link to Github issue: {GH link}

Reviewer: ```suggestion
     Link to GitHub issue: {GH link}
```

## Cleanup

> - [ ] In the Backoffice, delete the created Destination Target and Customer.

Reviewer: ```suggestion
- [ ] In back-office, delete the created Destination Target and Customer.
```
